.github/CODEOWNERS

```diff
@@ -145,3 +145,19 @@ sycl/include/sycl/ext/oneapi/bindless* @intel/bindless-images-reviewers
 sycl/source/detail/bindless* @intel/bindless-images-reviewers
 sycl/plugins/unified_runtime/ur/adapters/**/image.* @intel/bindless-images-reviewers
 sycl/test-e2e/bindless_images/ @intel/bindless-images-reviewers
```

Comment:
Think the comment must be above Line 147?

Reply:
This line belongs to the bindless images section, it just happens to be at the end of the list and right before the e2e test section